docs/getting-started/connect-server-to-github.md

```diff
@@ -18,22 +18,22 @@ When using the hosted service on https://app.amplication.com, the integration is
 ### Create a new GitHub App
 
 1. Go to https://github.com/settings/developers.
-2. Click on **OAuth Apps**.
-3. Click on **Register a new application**.
+2. Click **OAuth Apps**.
+3. Click **Register a new application**.
```

Comment:
This is not up to date. We are using a new method for sync with GitHub using "GitHub App" and not "OAuth App"
please talk with @morhag90 to get more details